src/chainbin/nnet3-chain-copy-egs.cc

```diff
@@ -224,41 +267,33 @@ void CalculateFrameSubsamplingFactor(const NnetChainExample &eg,
 void ModifyChainExampleContext(int32 left_context,
                                int32 right_context,
                                const int32 frame_subsampling_factor,
-                               NnetChainExample *eg) {
-  static bool warned_left = false, warned_right = false;
+                               NnetChainExample *eg,
```

Comment:
It's not clear to me what this function is supposed to do (or, actually, what it originally did).  Please add documentation!